Document gRPC stram message's `metadata` (#1735)
grafana/k6-docs: docs/sources/next/javascript-api/k6-net-grpc/stream/message-metadata.md

---

> title: 'Message Metadata'
> description: 'An object with the metadata of a message from gRPC stream.'
> weight: 50
> ---
> 
> # Message Metadata

Reviewer: ```suggestion
title: 'Metadata'
description: 'The metadata of a gRPC stream's message.'
weight: 50
---

# Metadata
```

Author: I've applied this locally since the `'` inside the description text breaks the YAML

> Is the metadata object of a gRPC stream's message.

Reviewer: ```suggestion
The `Metadata` is an object that represents the gRPC stream's message.
```

| Name          | Type   | Description                                                                                   |
| ------------- | ------ | --------------------------------------------------------------------------------------------- |

> | `Metadata.ts` | number | Contains the timestamp of the original event (for example, when a message has been received). |

Reviewer: ```suggestion
| `Metadata.ts` | number | Contains the timestamp of the original event. For example, when a message has been received. |
```
